src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -256,12 +256,6 @@ func (r *Cluster) validateKafkaListeners() field.ErrorList {
 						"only one kafka api listener can be marked as external"))
 			}
 			external = &r.Spec.Configuration.KafkaAPI[i]
```

Comment:
because it's not needed right now, it would require more tests etc. I can do it as a follow-up if we want to. I was just trying to deliver this ASAP :D

Reply:
also right now we cannot do it for schema registry unfortunately :(